src/EnergyPlus/WindowManager.hh

```diff
@@ -277,27 +279,12 @@ namespace Window {
     Real64 InterpolateBetweenFourValues(
         Real64 X, Real64 Y, Real64 X1, Real64 X2, Real64 Y1, Real64 Y2, Real64 Fx1y1, Real64 Fx1y2, Real64 Fx2y1, Real64 Fx2y2);
 
-    void W5LsqFit(Array1S<Real64> IndepVar, // Independent variables
-                  Array1S<Real64> DepVar,   // Dependent variables
-                  int N,                    // Order of polynomial
-                  int N1,                   // First and last data points used
-                  int N2,
-                  Array1S<Real64> CoeffsCurve // Polynomial coeffients from fit
-    );
-
-    void W5LsqFit2(Array1A<Real64> IndepVar, // Independent variables
-                   Array1A<Real64> DepVar,   // Dependent variables
-                   int N,                    // Order of polynomial
-                   int N1,                   // First and last data points used
-                   int N2,
-                   Array1A<Real64> CoeffsCurve // Polynomial coeffients from fit
+    void W5LsqFit(std::array<Real64, numPhis> const &ivars, // Independent variables
```

Comment:
Version of this function using `std::array`. Array size and polynomial order arguments removed because they were always the same.